graphql_compiler/query_formatting/common.py

```diff
@@ -34,47 +34,43 @@ def validate_argument_type(expected_type, value, name=None, is_inner_value=False
         value: object that can be interpreted as being of that type
         name: optional string. If it is a string, it is the name of the argument and will be used to
               provide more descriptive error messages.
-        is_inner_value: bool, whether this value corresponds to an element in a list argument.
     """
-    # TODO(pmantica1): make name parameter required in major version bump.
-    if name is not None:
-        if is_inner_value:
-            error_message_prefix = u'Invalid type for element in list argument {}.'.format(name)
-        else:
+    def _get_invalid_type_error_message(expected_python_type_name):
```

Comment:
Why put this function definition right here, as opposed to making it a private function at module scope? Leaking `name` in by accident like this seems unnecessary and confusing.